app/assets/javascripts/index.js

```diff
@@ -0,0 +1,35 @@
+var id = ''
+var buttons = "<button class='delete'>" + 'Delete' + "</button>"
++ "<button class='thumbs down'>" + 'thumbs down' + "</button>"
++ "<button class='thumbs up'>" + 'thumbs up' + "</button>"
+
+function renderIdeas(ideas) {
+  $.each(ideas, function(index, idea){
+    $('#ideas').append(
+      "<div class='each-idea' id='" + idea.id + "'><div class='edit' contenteditable='true'><h3>"
```

Comment:
lines 8 and 30 might be a time to consider setting that ideas class equal to a var, like `let $ideas = $('#ideas') but I also know that might cause problems if you've saved a DOM element that gets changed and later try to access its children